docs/tutorial/security.md

```diff
@@ -249,15 +249,15 @@ prevent the use of Node primitives, `contextIsolation` must also be used.
 
 ### Why?
 
-Context isolation allows each the scripts on running in the renderer to make
-changes to its JavaScript environment without worrying about conflicting with
-the scripts in the Electron API or the preload script.
+Context isolation allows scripts on running in the renderer to make changes to
+their JavaScript environment without worrying about conflicting with the scripts
+in the Electron API or the preload script.
 
 While still an experimental Electron feature, context isolation adds an
 additional layer of security. It creates a new JavaScript world for Electron
 APIs and preload scripts, which mitigates so-called "Prototype Pollution" attacks.
 
-At the same time, preload scripts still have access to the  `document` and
+At the same time, preload scripts still have access to their own  `document` and
 `window` objects. In other words, you're getting a decent return on a likely
 very small investment.
```

Comment:
Side note: I'm not sure if "a likely very small investment" is entirely accurate given that you could've relied heavily on exposing globals to the loaded script 😅